code/modules/mob/living/simple_animal/friendly/farm/animals/chicken.dm

```diff
@@ -0,0 +1,66 @@
+/mob/living/simple_animal/farm/chicken
+	name = "\improper chicken"
+	desc = "Hopefully the eggs are good this season."
+	icon_state = "chicken_white"
+	icon_living = "chicken_white"
+	icon_dead = "chicken_white_dead"
+	speak = list("Cluck!","BWAAAAARK BWAK BWAK BWAK!","Bwaak bwak.")
```

Comment:
Add cluck cluck cluck, tuck tuck, ERR ERR ERR, and  Kuh-kuh-kuh-KACK! 

Source: Have chickens